osu.Game.Rulesets.Osu/Mods/OsuModRepel.cs

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System;
+using System.Linq;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics.Sprites;
+using osu.Game.Configuration;
+using osu.Game.Rulesets.Osu.Utils;
+using osu.Game.Rulesets.Osu.Objects;
+using osu.Game.Rulesets.Osu.UI;
+using osuTK;
+
+namespace osu.Game.Rulesets.Osu.Mods
+{
+    internal class OsuModRepel : OsuEaseHitObjectPositionsMod
+    {
+        public override string Name => "Repel";
+        public override string Acronym => "RP";
+        public override IconUsage? Icon => FontAwesome.Solid.ExpandArrowsAlt;
```

Comment:
Please don't specify mod icons. We've decided to avoid doing so for the time being. Icons will be designed for mods specifically and properly at a later date, rather than picking the least worst option from fontawesome selection.